diff --git a/node_modules/expo-modules-core/android/build.gradle b/node_modules/expo-modules-core/android/build.gradle
--- a/node_modules/expo-modules-core/android/build.gradle
+++ b/node_modules/expo-modules-core/android/build.gradle
@@ -34,10 +34,12 @@
   }
 
   repositories {
+    google()
     mavenCentral()
   }
 
   dependencies {
+    classpath("com.android.tools.build:gradle:8.1.1")
     classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${getKotlinVersion()}")
   }
 }
